### PR TITLE
upgrade(installer): Add installer commands to instrument / uninstrument host & docker

### PR DESCRIPTION
**Pull request by BaptisteFoy**
### What does this PR do?
- Refactors a bit the injector setup to extract the `Instrument` & `Uninstrument` methods
- Create `datadog-installer` commands for these methods

<ins>In the next PR</ins> I'll make the install replace the deb injector install scripts to execute these new commands instead.

### Additional Notes
Corresponding system-tests PR (needs an installer staging release): TODO

### CODE REVIEW

File: pkg/fleet/installer/service/file.go [hunk 1]
```diff
@@ -20,6 +20,8 @@ import (
 	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer"
 )
 
+var rollbackNoop = func() error { return nil }
```
Comment (BaptisteFoy) [inline pkg/fleet/installer/service/file.go:23]: This is to make sure we always return something when there's no error and avoid accidental panics due to non-nil checking.

File: pkg/fleet/installer/service/apm_inject.go [hunk 2]
```diff
@@ -62,12 +82,35 @@ type apmInjectorInstaller struct {
 	ldPreloadFileUninstrument *fileMutator
 	dockerConfigInstrument    *fileMutator
 	dockerConfigUninstrument  *fileMutator
+	envs                      *env.Env
 }
 
 // Setup sets up the APM injector
-func (a *apmInjectorInstaller) Setup(ctx context.Context) (retErr error) {
+func (a *apmInjectorInstaller) Setup(ctx context.Context) error {
+	err := a.Instrument(ctx)
```
Comment (raphaelgavache) [inline pkg/fleet/installer/service/apm_inject.go:90]: either set Instrument at the end or propagate the rollback, or you'll fail installation and still instrument!

Comment (BaptisteFoy) [reply pkg/fleet/installer/service/apm_inject.go]: 👍 Done

**Review by raphaelgavache — APPROVED** [review]
nice, lgtm

**Pull request merged**